keyboards/3.0/bn.xml

```diff
@@ -0,0 +1,155 @@
+        <!-- exclam, etc -->
+
+        <!-- gap -->
+        <key id="ña" output="ঞ" />
+        <key id="ai" output="\u{09C8}" />
+        <key id="ṛ" output="\u{09C3}" />
+        <key id="tha" output="থ" />
+        <key id="ẏa" output="য়" /> <!-- Missing in Keyman version of file-->
+        <key id="ū" output="\u{09C2}" />
+        <key id="ī" output="\u{09C0}" />
+        <key id="au" output="\u{09CC}" />
+        <key id="pha" output="ফ" />
+
+        <key id="au-length" output="\u{09D7}" /> <!-- TODO: better name? -->
+        <key id="sha" output="ষ" />
+        <key id="dha" output="ধ" />
+        <key id="ṭha" output="ঠ" />
+        <key id="gha" output="ঘ" />
+        <!-- gap -->
+        <key id="jha" output="ঝ" />
+        <key id="kha" output="খ" />
+        <!-- gap -->
+
+        <!-- gap -->
+        <key id="vis-hasant" output="\m{X}" />
+        <key id="cha" output="ছ" />
+        <key id="ḍha" output="ঢ" />
+        <key id="bha" output="ভ" />
+        <key id="ṅa" output="ণ" />
+        <key id="nga" output="ঙ" />
+        <!-- less-than -->
+        <key id="nukta" output="\u{09BC}" />
+        <!-- question -->
+
+
+    </keys>
+    <layers formId="us">
+        <layer modifiers="none">
+            <row keys="n̐ 1 2 3 4 5 6 7 8 9 0 hyphen equal" />
+            <row keys="more wa e ra ta sha u i o pa open-square close-square backslash" />
+            <row keys="ā sa da ṭa ga ha ja ka la semi-colon apos" />
+            <row keys="śa hasant ca ḍa ba na ma comma dahri slash" />
+            <row keys="space" />
+        </layer>
+        <layer modifiers="shift">
+            <row
+                keys="ṃ bang at hash dollar percent caret amp asterisk open-paren close-paren underscore plus" />
+            <row keys="gap ña ai ṛ tha ẏa ū ī au pha open-curly close-curly pipe" />
+            <row keys="au-length sha dha ṭha gha gap jha kha gap colon double-quote" />
+            <row keys="gap vis-hasant cha ḍha bha ṅa nga open-angle nukta question" />
+            <row keys="space" />
+        </layer>
+    </layers>
+
+    <transforms type="simple">
+        <transformGroup>
+            <!-- these suport the 'more' key on q -->
+            <transform from="\u{q}:" to="\u{0983}" />
+            <transform from="\u{q}L" to="ৡ" />
+            <transform from="\u{q}।" to="॥" />
+            <transform from="\u{q}ড" to="ড়" />
+            <transform from="\u{q}ঢ" to="ঢ়" />
+            <transform from="\u{q}ত" to="ৎ" />
+            <transform from="\u{q}য" to="য়" />
+            <transform from="\u{q}র" to="ৰ" />
+            <transform from="\u{q}ল" to="ঌ" />
+            <transform from="\u{q}\u{09BE}" to="অ" />
+            <transform from="\u{q}\u{09BF}" to="ই" />
+            <transform from="\u{q}\u{09C0}" to="ঈ" />
+            <transform from="\u{q}\u{09C1}" to="উ" />
+            <transform from="\u{q}\u{09C2}" to="ঊ" />
+            <transform from="\u{q}\u{09C3}" to="ৠ" />
```

Comment:
This is a long one (RR), are you sure you don't want short one (98B R)?  there is no long vowel sign (9C4)

Reply:
This matches the original keyboard at https://help.keyman.com/keyboard/sil_bengali_phonetic/1.2.1/sil_bengali_phonetic - q e = U+098F